static-pages/questions.md

```diff
@@ -0,0 +1,28 @@
+title: Have a question? We're here to help!
+
+<div class="help-center-cards text-white row my-default">
+<div class="col-sm-4">
+    <div class="help-center-card card text center mx-default">
+      <div class="card-body pxy-default">
+        <h3 class="card-title"><a href="/help-center/">Help Center &raquo;</a></h3>
+        <h5 class="card-text">Find guides to help you tag, transcribe, and review.</h5>
+      </div>
+    </div>
+  </div>
+  <div class="col-sm-4">
+    <div class="help-center-card card text center mx-default">
+      <div class="card-body pxy-default">
+        <h4 class="card-title"><a target="_blank" href="https://historyhub.history.gov/community/make-your-mark/">Discuss &raquo;</a></h4>
```

Comment:
These = signs look really weird, are they correct?

Reply:
I think that's GitHub's parser choking on HTML-in-Markdown

Reply:
The generated markup is valid and the resulting page looks correct, too.